deploy/template.yaml

```
@@ -4316,10 +4326,24 @@ Resources:
     Type: AWS::Serverless::StateMachine
     Properties:
       Name: !Sub "F2F-YotiCallback-stepfunction-${AWS::StackName}"
+      AutoPublishAlias: live
+      DeploymentPreference:
+        Type: !Ref StepFunctionsDeploymentPreference
+        Interval: !If [IsProdEnvironment, 15, 5]
+        Percentage: !If [IsProdEnvironment, 10, 50]
+        Alarms: !If
+          -UseStepCanaryDeploymentAlarms
```

Does there need to be a space here?
```suggestion
          - UseStepCanaryDeploymentAlarms
```